scripts/apps/archive/directives/MarkedItemTitle.js

```diff
@@ -57,7 +57,7 @@ class LinkFunction {
         this.scope.$watch('item', (item) => {
             if (item) {
                 this.scope.marks = [];
-                if (item[this.scope.markField].length) {
+                if (!_.isEmpty(item[this.scope.markField])) {
```

Comment:
@pavlovicnemanja has also made a fix for this line: 
https://github.com/superdesk/superdesk-client-core/pull/1700/files#diff-7ffbaf915ec369a4c5eb37a2ad9cca95R60